code/datums/antagonists/datum_cult.dm

```diff
@@ -5,13 +5,20 @@
 	qdel(communion)
 	return ..()
 
+/datum/antagonist/cult/can_be_owned(datum/mind/new_owner)
+	. = ..()
+	if(.)
+		. = is_convertable_to_cult(new_owner.current)
+
 /datum/antagonist/cult/on_gain()
 	. = ..()
-	if(!owner)
-		return
+	SSticker.mode.cult += owner
+	SSticker.mode.update_cult_icons_added(owner)
```

Comment:
Yea I did, I copied this from another branch that I had removed ticker procs and completly forgot about them